src/routes/dashboard/getMyEvents.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const router = express.Router();
+const mysql = require('mysql');
+const config = require('../../../config.js');
+
+// get 5 upcoming events the user has RSVPed to
+router.post('/api/getMyEvents', (req, res) => {
```

Comment:
since you're using a search filter, you have to pass in a query instead of a body. Refer to Gabby/Esha's code on how to do a filtered search with a `GET` API

Reply:
updated thanks